### ISSUE TITLE
Country page: Highlight member country

### ISSUE THREAD

**Issue by ramyaragupathy** [hotosm/hotosm-website#362]
Member countries are highlighted with blue strokes in landing page. However on choosing a member country, strokes theme change to red in country-page. Should we keep it similar to our-work page? I'm just looking at ways to differentiate a project/member in country-page. This could be with styling or the sidebar text.

![country-page](https://user-images.githubusercontent.com/12103383/45498580-f305fa00-b797-11e8-82c4-dd7f72407d06.gif)

**Comment by 5anjana** [comment]
Hii @ramyaragupathy  I would like to work on this issue. Can you assign it to me?

**Comment by ramyaragupathy** [comment]
Feel free to pick this @sanjana11147. Leave a comment here if you're going to work on this

**Comment by 5anjana** [comment]
Sure. I will work on it.

**Comment by bismitaguha** [comment]
Hi @ramyaragupathy . I am giving a try to this issue too.

**Comment by 5anjana** [comment]
Yeah. I am not able to work on it currently so you should take it up @codefab99

**Comment by bismitaguha** [comment]
Ok I am working on it.

**Comment by nav-p** [comment]
Can i work on this issue @ramyaragupathy

**Comment by bismitaguha** [comment]
@na899 please find another issue. I am already working on this.
